sopel/plugins/handlers.py

```diff
@@ -128,47 +130,47 @@ def is_loaded(self):
         raise NotImplementedError
 
     def setup(self, bot):
-        """Setup the plugin with the ``bot``
+        """Run the plugin's setup routine.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def has_setup(self):
-        """Tell if the plugin has a setup action
+        """Tell if the plugin has a setup routine.
```

Comment:
Ditto my comment explaining the suggestion on `setup()`.

```suggestion
        """Tell if the plugin has a setup action.
```